### DISCUSSION TITLE
Configure validator log level

### DISCUSSION THREAD

**Discussion by harto** [MobilityData/gtfs-validator#1251] [Q&A]
Hello, I was wondering how to control the log level of the validator CLI.

Right now it appears to log `INFO` messages, including a bunch of information about cache utilisation, etc. I would prefer to only see warnings and errors in the CLI output.

I understand it may be possible by setting some Java system property, but I'm not exactly sure which one(s) ☺️

**Comment by isabelle-dr** [comment]
Hello,

I agree, that having the ability to filter some of the CLI output would be helpful. 
This issue was discussed in issue #1194, and it seems like the solution would be to implement issue #256 in this project (a verbosity option).
Is this something you would be interested in helping with?

**Reply by harto** [reply]
Ah, I thought perhaps there might be a workaround using a Java system property, e.g. `java -Dsomething=something -jar …`, but I don't know enough about Java logging infrastructure to figure it out.

In principle I would be glad to contribute, but in reality I don't have any time to spare.

**Reply by isabelle-dr** [reply]
I know the feeling 🥲.
